docs/introduction/install-superface.mdx

```diff
@@ -0,0 +1,41 @@
+# Install Superface
+
+Our CLI is a command-line utility that acts as your starting point for working with Superface, from creating your account through all the steps to get your first API use case up and running.
+
+## macOS/Linux
+
+If you have the [Homebrew](https://brew.sh) package manager install, Superface CLI can be install by running:
+
+```bash
+brew install superfaceai/cli/superface
+```
+
+If not, you can install it using [NPM](https://npmjs.org) by running:
+
+```bash
+npm install -g @superfaceai/cli@latest
+```
+
+## Windows
+
+On Windows, you can install it using [NPM](https://npmjs.org) by running:
+
+```bash
+npm install -g @superfaceai/cli@latest
+```
+
+Alternatively, if using Windows Subsystem for Linux (WSL), you can also install [Homebrew](https://brew.sh/) and install the CLI using that.
+
+## After installing
+
+Once installed, you will be able to use the CLI directly from your command-line by running either the `superface` or `sf` commands.
+
+To get started, you can run either of the following commands:
+
+```bash
+superface --help
+```
+
+```bash
+superface login
```

Comment:
Maybe worth adding a sentence "to use the CLI you need Superface account, you can create the account by signig up at superface.ai or running sf login"...